### ISSUE TITLE
three.js documentation Chinese version

### ISSUE THREAD

**Issue by nicholas-yangding** [mrdoob/three.js#14748]
Hi, We are a team from china who love open source.We want to translate the documentation into 
Chinese. About now our plan is to translate pages and convert them into static pages. 
So I want to know how we can put these pages into this repo , or merge them into the threejs.org documentation part?
Pleased to hear reply.

**Comment by mrdoob** [comment]
Previous discussion: #6840

The idea was to move all [these files](https://github.com/mrdoob/three.js/tree/dev/docs/api) inside a `en` folder, copy the folder into a `zh` folder and start translating one by one.

**Comment by mrdoob** [comment]
Okay, I just did added basic support: #14750

**Comment by mrdoob** [comment]
If you copy the folder `api/en/` to `api/zh/` these links will work:

https://rawgit.com/mrdoob/three.js/dev/docs/#api/en/animation/AnimationAction
https://rawgit.com/mrdoob/three.js/dev/docs/#api/zh/animation/AnimationAction

**Comment by mrdoob** [comment]
Things will definitely be out of sync for a while but I think we should start somewhere.

**Comment by takahirox** [comment]
OK

> Things will definitely be out of sync for a while

Let's add this note somewhere. Maybe it's good start.

**Comment by mrdoob** [comment]
We still need to add a language selector in the left panel somehow.

In the meantime, @nicholas-yangding do you think you can start adding pages?

**Comment by WestLangley** [comment]
>https://rawgit.com/mrdoob/three.js/dev/docs/#api/en/animation/AnimationAction

Won't that break existing links to the docs in help sites, for example?

**Comment by nicholas-yangding** [comment]
@mrdoob ok, thanks so much. We will start it next week because this week I need to gather some people. 
Besides, Not only the api docs we will translate, other docs too. Then Other docs how to organize them?

**Comment by mrdoob** [comment]
We can do the same for manual. Do you want to start translating the manual first?

**Comment by nicholas-yangding** [comment]
@mrdoob yes thanks. We have about 20+ engineers, and we want cover all of the website.

**Comment by chenxiaoleizi** [comment]
@nicholas-yangding 我能加入翻译吗？
good idea!

**Comment by nicholas-yangding** [comment]
@mrdoob so much thanks!  Maybe a language filter should be added on the left in page? We will start soon after this week
@HypnosNova @chenxiaoleizi We will have a wechat group to communicate with each other. Please send me(nicholaswang101@gmail.com) an email with your wechat id. So pleased to see so many people participated!

**Comment by sweerwen** [comment]
@nicholas-yangding  count me in

**Comment by nicholas-yangding** [comment]
@mrdoob now the translation is in progress, soon we will pull first request to the repo with some translations.

**Comment by HypnosNova** [comment]
@nicholas-yangding Have you invited me to the wechat group?

**Comment by nicholas-yangding** [comment]
@mrdoob hi, please add /zh to this directory 
![image](https://user-images.githubusercontent.com/5335686/45938385-00bf4900-bffc-11e8-9635-17768a11d74e.png)
files in it also need to be translated 
so much thx!

**Comment by WestLangley** [comment]
>please add /zh to this directory to examples

If you are going to do that, how about a separate repo, instead? Maybe a separate repo for both the zh docs and zh examples?

**Comment by mrdoob** [comment]
@WestLangley I don't think they mean to the examples folder. I think they mean to the examples section in the docs.

**Comment by Mugen87** [comment]
> I think they mean to the examples section in the docs.

Done with #16680 🎉

**Comment by linbingquan** [comment]
relate https://github.com/mrdoob/three.js/pull/19228

**Comment by Sooia** [comment]
Hello, I am a Chinese native speaker, quite interested in Three.js development. And I want to join in the Chinese version of three.js documentation. How can I participate in?

**Comment by Mugen87** [comment]
@Sooia There are some pages in the `examples` section which are not yet translated. E.g.:

https://threejs.org/docs/index.html#examples/zh/loaders/SVGLoader
https://threejs.org/docs/index.html#examples/zh/utils/BufferGeometryUtils
https://threejs.org/docs/index.html#examples/zh/utils/SceneUtils

Translating these pages would be a great contribution.

**Comment by Sooia** [comment]
> @Sooia There are some pages in the `examples` section which are not yet translated. E.g.:
> 
> https://threejs.org/docs/index.html#examples/zh/loaders/SVGLoader https://threejs.org/docs/index.html#examples/zh/utils/BufferGeometryUtils https://threejs.org/docs/index.html#examples/zh/utils/SceneUtils
> 
> Translating these pages would be a great contribution.

Hi, @Mugen87 . I have translated the pages in the examples that you mentioned above, and I have created a PR #23262. Please have a check. Thanks. If there are other pages that haven't been translated, please @ me or email me( faaa@live.com ) to let me know.

**Comment by cswwww** [comment]
@Mugen87 Hello, I am a developer from China. I'm learning threeJS and noticed some differences between the Chinese and English manuals since PR #25468. I would like to know if this translation and synchronization task has already been picked up, and if not, I would like to participate in it. 
I plan to update the Chinese manual by translating from the English manual.

**Comment by Mugen87** [comment]
> I would like to know if this translation and synchronization task has already been picked up

Not that I am aware of.

**Comment by Mugen87** [comment]
With `r175` there will be a new documentation based on JSDoc. The new docs cover the entire code base (core+addons) and support new features like client-side search. However, since the documentation is generated via JSDoc, there is no i18n support. The pages are only available in English.

Translated versions of the documentation are not maintained in this repository anymore, see #24984.
